io.github.nest-desktop.yml

```diff
@@ -0,0 +1,88 @@
+id: io.github.nest-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+  - org.freedesktop.Sdk.Extension.typescript
+command: nest-desktop-launch
+separate-locales: false
+rename-desktop-file: nest-desktop.desktop
+rename-icon: nest-desktop
+rename-appdata: nest-desktop
+finish-args:
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to save files locally
+  - --filesystem=home
```

Comment:
does this work with filechooser portals?

Reply:
Yes, it works with filechooser dialog.

Reply:
why is `home` still needed?

Reply:
I am testing it without `home`.

Reply:
Without `home` the filechooser doesn't work well. 

I need it to export data to json file or import data from json file.